tools/scrape_supported_devices.py

```diff
@@ -41,10 +41,13 @@ def getmarkdownheader():
   """Get the generated header
   """
   srctime = getgitcommittime()
+  # pylint: disable=C0209
   return """<!--- WARNING: Do NOT edit this file directly.
       It is generated by './tools/scrape_supported_devices.py'.
       Last generated: {} --->""".format(
           time.strftime("%a %d %b %Y %H:%M:%S +0000", time.gmtime(srctime)))
```

Comment:
```suggestion
  return f"""<!--- WARNING: Do NOT edit this file directly.
      It is generated by './tools/scrape_supported_devices.py'.
      Last generated: {time.strftime("%a %d %b %Y %H:%M:%S +0000", time.gmtime(srctime))} --->"""
```

Reply:
I actually tried this one, but it hit the 80-char linter limit, and f""" """ hates a newline being added so I left it.